core/dbt/task/deps.py

```diff
@@ -56,19 +56,36 @@ def run(self):
                 self.config, self.config.cli_vars
             ))
 
+            packages_to_upgrade = []
             for package in final_deps:
+                package_name = package.name
+                source_type = package.source_type()
+                version = package.get_version()
+
                 logger.info('Installing {}', package)
                 package.install(self.config, renderer)
                 logger.info('  Installed from {}',
                             package.nice_version_name())
+                if source_type == 'hub':
+                    version_latest = package.get_version_latest()
+                    if version_latest != version:
+                        packages_to_upgrade.append(package_name)
+                        logger.info('  Updated version available: {}',
+                                    version_latest)
+                    elif version_latest == version:
```

Comment:
Nit: this can be an `else` since there is no other possible state (it can't be not equal to `version` and equal to `version`

Reply:
Great call out! Made the change :)